src/components/common/Tree/Tree.js

```diff
@@ -34,7 +34,9 @@ const DynamicTreeView = ({
   const onToggle = (_event, nodeIds) => setExpandedItems(nodeIds);
 
   // show only folder items in the navigation tree
-  const itemsFiltered = items.filter((item) => item.type === ITEM_TYPES.FOLDER);
+  const itemsFiltered = items.filter((item) =>
+    [ITEM_TYPES.FOLDER, ITEM_TYPES.SHORTCUT].includes(item.type),
```

Comment:
Same nitpick as before. Since you re-use the exact same array in both this component and in the previous `CustomItemTree`, it would probably make sense to extract this array into a constant.